adcc/backends/psi4.py

```diff
@@ -37,20 +39,29 @@ def __init__(self, wfn):
         self.wfn = wfn
         self.backend = "psi4"
         self.mints = psi4.core.MintsHelper(self.wfn)
+        warnings.warn("Gauge origin selection not available in "
```

Comment:
Are these warnings really necessary? I think the `NotImplementedError` once a different gauge origin is selected, should be sufficient.